ArchiSteamFarm/ArchiSteamFarm.csproj

```diff
@@ -69,6 +69,7 @@
     <PackageReference Include="Swashbuckle.AspNetCore.Annotations" Version="5.0.0" />
     <PackageReference Include="Swashbuckle.AspNetCore.Newtonsoft" Version="5.0.0" />
     <PackageReference Include="System.Composition" Version="1.4.0" />
+    <PackageReference Include="System.Linq.Async" Version="4.0.0" />
```

Comment:
What do we need this for? I thought `IAsyncEnumerable` is in .NET Core 3.1, no?

Reply:
Yes, but this package is required for making LINQ calls to the `IAsyncEnumerable`.